ui/page/listeners.go

```diff
@@ -179,6 +179,8 @@ func (mp *MainPage) UpdateNotification(signal interface{}) {
 }
 
 func (mp *MainPage) desktopNotifier(notifier interface{}) {
+	//proposalNotification:= l.WL.Wallet.ReadBoolConfigValueForKey("proposalnotificationkey")
+	proposalNotification := mp.WL.Wallet.ReadBoolConfigValueForKey("proposalnotificationkey")
```

Comment:
`proposalnotificationkey` move this to load package utils.go file.

```suggestion
	proposalNotification := mp.WL.Wallet.ReadBoolConfigValueForKey(load.ProposalNotificationConfigkey)
```